crates/rethnet_rpc_server/tests/integration_tests.rs

```diff
@@ -29,13 +35,15 @@ async fn start_server() -> SocketAddr {
         },
     );
 
-    let server = Server::new(
-        Config {
-            address: "127.0.0.1:0".parse::<SocketAddr>().unwrap(),
-            rpc_hardhat_network_config: RpcHardhatNetworkConfig { forking: None },
-        },
-        accounts,
-    )
+    let server = Server::new(Config {
+        address: "127.0.0.1:0".parse::<SocketAddr>().unwrap(),
```

Comment:
Always opt for static guarantees when possible, in this case by just constructing a `SocketAddr` using its constructor

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4149/commits/b68b5046b2f6981ee2023c24b31bdcfa89ca6e82